tests-ci/tests/test_pxColorNames.js

```diff
@@ -62,6 +62,13 @@ var tests = {
             results.push(assert(rect.fillColor === value," fillColor: '"+color+"' ("+value+") != " + rect.fillColor ));
         });
 
+        resolve(results);
```

Comment:
Remove this resolve in favor of the one that will happen regardless of promise resolve or reject.  Do the same for all tests, please.

Reply:
done